pages/static/vendor/js/djaodjin-sidebar-gallery.js

```diff
@@ -21,16 +21,29 @@
         return modal;
     };
 
+    var singleMedia = function(file, index){
+        var single_media = '';
+        if (file.file_src.indexOf('.mp4') > 0){
```

Comment:
It should be based on a content-type field passed back by the API.  Server-side, [python-magic](https://github.com/ahupp/python-magic) can be use to guess the type of a file.